### PR TITLE
PopUp

### PR DESCRIPTION
**Pull request by isaka-lumato**
## To add the popup window with all Information it requires dynamically here are what was done
 
- [ ] Create An object that will contain all information needed in the popup window
- [ ]  Put the object inside our array which is our collection of data.
- [ ]  implement the fex card project in HTML dynamically with data coming from the array.
- [ ]  implement our Js file to check how to render the popup window
- [ ]  add data to the popup window dynamically with data coming in the same array;
- [ ]  syle the popup window in my CSS file
- [ ]  Add some on click event to the s See Project button to show the menu;
- [ ]  correct linter error sin my HTML, CSS, and JS file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -269,8 +273,27 @@ <h2>Contact me</h2>
...
+        <div class="modalMiddle">
+          <div class="modalTechnology"></div>
+          <div class="modalFooter">
+            <button class="source"><span class="sourceImg"><img src="" alt="image1" class="img1"></span></button>
+            <button class="life"><span class="lifeImg"><img src="" alt="image2" class="img2"></span></button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div> -->
```
Comment (Dipeshtwis) [inline index.html]: - [ ] Remove all the commented code from your project since it makes code not readable.

File: app.js [hunk 2]
```diff
@@ -18,3 +18,130 @@ menuLink.forEach((link) => {
...
+        <div class="popup-body">
+            <p>${project.paraText}</p>
+            <div class="pop-buttons">
+                <ul class="progrm-l">
+                ${project.languageList.map((link) => ` <li class="pop-lang">${link}</li>`).join('')}               
+                </ul>
+                <div class="pop-hr"></div>
+                <div class="see-popup">
+                    <a href="#"><span>${project.button1Text}</span><img src="Icon - Export.svg" alt="close3"></a>
+                     <a href="#"><span>${project.button2Text}</span><img src="Icon - Export.svg" alt="close3"></a>
```
Comment (Dipeshtwis) [inline app.js:119]: - [ ] This two button should have some margin in between in the desktop version, for now your button has no gap. Please follow the design.

### Your design


Screenshot from 2021-07-04 00-48-14


### Figma design

Screenshot from 2021-07-04 00-48-25

File: app.js [hunk 3]
```diff
@@ -18,3 +18,130 @@ menuLink.forEach((link) => {
...
+  id: 3,
+  Heading: 'TONIC',
+  imageUrl: 'imagesdesktop/tonic2-3.png',
+  paraText: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industry s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been theindustrys standard dummy text ever since the 1500s,when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languageList: ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'],
+  button1Text: 'see live',
+  button2Text: 'see source',
+},
+{
+  id: 4,
```
Comment (Dipeshtwis) [inline app.js:50]: [OPTIONAL]

- [ ] You should use custom text rather than `lorem ipsum` :blush:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @isaka-lumato, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of flexbox.
:+1: The projects section is created dynamically with the information from the projects array.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- I suggest that your pull request be more descriptive describing introduced changes in more details this is a recommended best practice as it makes reviewing easier.

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sinansevgi) [reply index.html]: :heavy_check_mark: Well Done

File: app.js (hunk 2 above)
Comment (sinansevgi) [reply app.js:119]: :heavy_check_mark: Well Done

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @isaka-lumato , 

Great work with making all the required changes by previous.
Your project is complete!  
Congratulations! 🎉


You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**